5-Cookies/server/tests/env/drivers/appDriver.ts

```diff
@@ -0,0 +1,63 @@
+import axios from "axios";
+
+export class AppDriver {
+  url:string;
+  userID:string;
+  constructor(url: string) {
+    this.url = url;
+  }
+
+  setUserCookie(userID: string) {
+    this.userID = userID;
+  }
+  clearUserCookie() {
+    this.userID = undefined;
+  }
+
+  async getTodos() {
+    return axios.get(
+      `${this.url}/todos`,
+      this.userID && {
+        headers: {
+          Cookie: `userID=${this.userID};`,
+        },
```

Comment:
repeated several times, export to function

Reply:
Done.